btree/btree.c

```diff
@@ -2748,15 +2909,24 @@ static int ff_count_rec(const struct nd *node)
 
 static int ff_space(const struct nd *node)
 {
-	struct ff_head *h = ff_data(node);
-	int             crc_size = 0;
+	struct ff_head *h          = ff_data(node);
+	uint32_t        used_space = 0;
+	int             crc_size   = 0;
 
 	if (h->ff_level == 0 &&
 	    ff_crctype_get(node) == M0_BCT_BTREE_ENC_RAW_HASH)
 		crc_size = CRC_VALUE_SIZE;
 
-	return h->ff_nsize - sizeof *h -
-		(h->ff_ksize + ff_valsize(node) + crc_size) * h->ff_used;
+	if (IS_EMBEDDED_INDIRECT(node)) {
+		used_space = sizeof(struct ff_head) +
+			     (h->ff_max_recs * sizeof(struct ff_dir_entry)) +
+			     (h->ff_used * (h->ff_ksize + ff_valsize(node) +
+					    crc_size));
+	} else
+		used_space = sizeof *h + ((h->ff_ksize + ff_valsize(node) +
```

Comment:
Code consistency: 
On line 2921 and 2926, use either sizeof *h or sizeof(struct ff_head)

Reply:
Done.